### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by waqasgulzar**

### CODE REVIEW

File: src/app/modules/bookingapps/bookingapps.component.ts [hunk 1]
```diff
@@ -8,9 +8,9 @@ import { Router } from "@angular/router";
 export class BookingappsComponent implements OnInit {
 
     constructor(private router: Router) {
-        if (sessionStorage.getItem("organizationId") == null) {
-            this.router.navigate(['']);
-        }
+        // if (sessionStorage.getItem("organizationId") == null) {
```
Comment (ghazanfarseyal) [inline src/app/modules/bookingapps/bookingapps.component.ts]: Remove it

File: src/app/modules/bookingapps/bookingapps.component.ts [hunk 2]
```diff
@@ -8,9 +8,9 @@ import { Router } from "@angular/router";
...
+        //     this.router.navigate(['']);
+        // }
```
Comment (ghazanfarseyal) [inline src/app/modules/bookingapps/bookingapps.component.ts]: remove these commented lines

File: src/app/modules/marketing/marketing.component.ts [hunk 3]
```diff
@@ -12,9 +12,9 @@ import { Router } from '@angular/router';
 })
 export class MarkettingComponent implements OnInit {
   constructor(private router: Router) {
-    if (sessionStorage.getItem('organizationId') == null) {
-      this.router.navigate(['']);
-    }
+    // if (sessionStorage.getItem('organizationId') == null) {
```
Comment (ghazanfarseyal) [inline src/app/modules/marketing/marketing.component.ts]: remove it too

File: src/app/modules/medialibrary/mediaLibrary.component.ts [hunk 4]
```diff
@@ -37,15 +37,16 @@ export class MediaLibraryComponent implements OnInit {
   ) {}
 
   ngOnInit() {
-    this.accesLevels = [
-      { id: 1, title: 'Personal' },
-      { id: 2, title: 'Local share' },
-      { id: 3, title: 'Global share' }
-    ];
+    // this.accesLevels = [
```
Comment (ghazanfarseyal) [inline src/app/modules/medialibrary/mediaLibrary.component.ts]: don't comment, remove it

File: src/app/modules/monthlyOverview/monthlyOverview.component.ts [hunk 5]
```diff
@@ -8,9 +8,9 @@ import { Router } from "@angular/router";
 export class MonthlyOverviewComponent implements OnInit {
 
     constructor(private router: Router) {
-        if (sessionStorage.getItem("organizationId") == null) {
-            this.router.navigate(['']);
-        }
+        // if (sessionStorage.getItem("organizationId") == null) {
+        //     this.router.navigate(['']);
+        // }
```
Comment (ghazanfarseyal) [inline src/app/modules/monthlyOverview/monthlyOverview.component.ts]: Remove it

File: src/app/modules/bookingapps/bookingapps.component.ts (hunk 1 above)
Comment (waqasgulzar) [reply src/app/modules/bookingapps/bookingapps.component.ts]: sure i will remove all commented code.

**Pull request merged**